declarations/Module.d.ts

```diff
@@ -0,0 +1,337 @@
+     * @param {Chunk=} ignoreChunk chunk to be ignored
+     * @returns {boolean} true, if the module is accessible from "chunkGroup" when ignoring "ignoreChunk"
+     */
+    isAccessibleInChunkGroup(chunkGraph: import("./ChunkGraph"), chunkGroup: import("./ChunkGroup"), ignoreChunk?: import("./Chunk")): boolean;
+    /**
+     * @param {Chunk} chunk a chunk
+     * @param {ModuleGraph} moduleGraph the module graph
+     * @param {ChunkGraph} chunkGraph the chunk graph
+     * @returns {boolean} true, if the module has any reason why "chunk" should be included
+     */
+    hasReasonForChunk(chunk: import("./Chunk"), moduleGraph: import("./ModuleGraph"), chunkGraph: import("./ChunkGraph")): boolean;
+    /**
+     * @param {ModuleGraph} moduleGraph the module graph
+     * @returns {boolean} true if at least one other module depends on this module
+     */
+    hasReasons(moduleGraph: import("./ModuleGraph")): boolean;
+    /**
+     * @param {ModuleGraph} moduleGraph the module graph
+     * @returns {boolean} true, if the module is used
+     */
+    isModuleUsed(moduleGraph: import("./ModuleGraph")): boolean;
+    /**
+     * @param {ModuleGraph} moduleGraph the module graph
+     * @param {string | string[]} exportName a name of an export
+     * @returns {UsageStateType} state of the export
+     */
+    isExportUsed(moduleGraph: import("./ModuleGraph"), exportName: string | string[]): 0 | 1 | 2 | 4 | 3;
+    /**
+     * @param {ModuleGraph} moduleGraph the module graph
+     * @param {string | string[]} exportName a name of an export
+     * @returns {string | string[] | false} false, when module or referenced export is unused.
+     *                           string, the mangled export name when used.
+     */
+    getUsedName(moduleGraph: import("./ModuleGraph"), exportName: string | string[]): string | false | string[];
+    /**
+     * @param {NeedBuildContext} context context info
+     * @param {function(WebpackError=, boolean=): void} callback callback function, returns true, if the module needs a rebuild
+     * @returns {void}
+     */
+    needBuild(context: NeedBuildContext, callback: (arg0?: import("./WebpackError"), arg1?: boolean) => void): void;
+    /**
+     * @deprecated Use needBuild instead
+     * @param {TODO} fileTimestamps timestamps of files
+     * @param {TODO} contextTimestamps timestamps of directories
+     * @returns {boolean} true, if the module needs a rebuild
+     */
+    needRebuild(fileTimestamps: any, contextTimestamps: any): boolean;
+    /**
+     * @param {Hash} hash the hash used to track dependencies
+     * @param {ChunkGraph} chunkGraph the chunk graph
+     * @returns {void}
+     */
+    updateHash(hash: import("./util/Hash"), chunkGraph?: import("./ChunkGraph")): void;
+    /**
+     * @returns {void}
+     */
+    invalidateBuild(): void;
+    /**
+     * @abstract
+     * @returns {string} a unique identifier of the module
+     */
+    identifier(): string;
```

Comment:
`@abstract` seem to be ignored

Reply:
It does not supported yet (= and ‘@implements’ too